### PR TITLE
Gather scripts

### PR DESCRIPTION
**Pull request by shayne-fletcher**
In this PR the v4/v5 update scripts and triggers are collated into one file `Automation.daml`.

### CODE REVIEW

File: upgrade-v4-v5/automation/daml/Automation.daml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+init : LedgerParties -> Script ()
+init LedgerParties {company} = do
+  employeeRoles <- fmap snd <$> query @V4.EmployeeRole company
+  F.forA_ employeeRoles $
+   \V4.EmployeeRole {employee, company, ..} -> do
+     existingProposal <- query @V4V5.UpgradeProposal employee
+     if null existingProposal
+       then do
+        void $ (submit company $ do
```
Comment (hurryabit) [inline upgrade-v4-v5/automation/daml/Automation.daml]: `void $ (...)`? That's like belt and braces. 😉
In fact, `dlint` warns about this. And below as well.

File: upgrade-v4-v5/automation/daml.yaml [hunk 2]
```diff
@@ -1,18 +1,20 @@
 sdk-version: 0.13.51
```
Comment (hurryabit) [inline upgrade-v4-v5/automation/daml.yaml]: We now have slightly nested DAML projects here. The folder structure look something like
```
upgrade-v4-v5
+- daml.yaml
+- daml
   +- ...
+- automation
   +- daml.yaml
   +- daml
      +- ...
```
We should make the two DAML projects siblings of each other at the directory level. My approach would be to move the automation as `upgrade-v4-v5-automation` to the top level.

File: upgrade-v4-v5/automation/daml/Automation.daml (hunk 1 above)
Comment (shayne-fletcher) [reply upgrade-v4-v5/automation/daml/Automation.daml]: 😊

File: upgrade-v4-v5/automation/daml.yaml (hunk 2 above)
Comment (shayne-fletcher) [reply upgrade-v4-v5/automation/daml.yaml]: Done

**Comment by shayne-fletcher** [comment]
@hurryabit Thanks for the comments. All resolved I think. Propose to land this imminently if you wanted a chance for a last look.

File: davl-v4-v5-upgrade.sh [hunk 3]
```diff
@@ -9,11 +9,36 @@ set -eou pipefail
...
+ (cd admin-cli && yarn workspaces run build)
 yarn run davl-admin-cli v4-init -f $test_setup
-(cd upgrade-v4-v5/scripts/upgrade-init&&./upgrade-init.sh $test_party)
-(cd upgrade-v4-v5/scripts/upgrade-accept&&./upgrade-accept.sh $test_party)
-(cd upgrade-v4-v5/scripts/upgrade-finish&&./upgrade-finish.sh $test_party)
+(cd upgrade-v4-v5-automation && \
+ daml build -o .daml/dist/automation.dar \
+)
+
+(cd upgrade-v4-v5-automation && \
```
Comment (hurryabit) [inline davl-v4-v5-upgrade.sh]: Why do we change into that directory so many times instead of staying just there. Let's simplify this in a separate PR.

**Pull request merged**